### ISSUE TITLE
Can't get AccessToken

### ISSUE THREAD

**Issue by the-jay** [Geosearchef/placebot#5]
code fails on:

data_str = BeautifulSoup(r.content, features="html.parser").find("script", {"id": "data"}).contents[0][len("window.__r = "):-1]

Error encountered while running bot: 'NoneType' object has no attribute 'contents'

**Comment by Geosearchef** [comment]
Seems like it's not finding the script tag with `id=data` in there. I'd take a look at r.content manually to debug in your case. Are you sure the login is working?

**Comment by the-jay** [comment]
Yeah, the r.content returned is a logged in page, with my user info. But there's no script tag with id data anywhere in the content.

**Comment by Geosearchef** [comment]
Are you using the old reddit layout? I think the script doesn't work then. Just switch to the new layout or feel free to adapt the code and create a pull request.
